maven/bnd-maven-plugin/src/main/java/aQute/bnd/maven/plugin/BndMavenPlugin.java

```diff
@@ -176,14 +179,6 @@ public void execute() throws MojoExecutionException, MojoFailureException {
 				throw new MojoExecutionException("Sub-bundles not permitted in a maven build");
 			}
 
-			// Reject wab projects
-			if (builder.getProperty(Constants.WAB) != null) {
-				throw new MojoExecutionException(Constants.WAB + " not supported in a maven build");
-			}
-			if (builder.getProperty(Constants.WABLIB) != null) {
-				throw new MojoExecutionException(Constants.WABLIB + " not supported in a maven build");
-			}
-
```

Comment:
I don't see how this will work since the wab instruction fundamental reorganize the arrangement of the bundle. Classes move from the root to web-inf, bundle class path is set. How will the bnd-maven-plugin do that given the the files already in place in  target/classes? I don't see any code here to rearrange target/classes into the wab shape.

Reply:
I see your comment about maven-war-plugin now. Perhaps then the check here should confirm those plugins are in use when a wab instruction is in effect?

Reply:
To my knowledge there isn't a reliable way to know this in maven.